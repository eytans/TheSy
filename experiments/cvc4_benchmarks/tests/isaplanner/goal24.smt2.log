19:28:44 [WARN] Collecting statistics
19:28:44 [WARN]  ex is: zero
19:28:44 [WARN]  ex is: (succ zero)
19:28:44 [WARN]  ex is: (succ (succ zero))
19:28:44 [WARN] Running exploration without proof mode, but goals were given
19:28:44 [WARN] Proof mode: true
19:28:44 [WARN]  ex is: zero
19:28:44 [WARN]  ex is: (succ zero)
19:28:44 [WARN]  ex is: (succ (succ zero))
19:28:44 [WARN] Running exploration without proof mode, but goals were given
19:28:44 [WARN] Proof mode: true
19:28:44 [WARN] proved: (nmax ?ts_ph_Nat_0 ?ts_ph_Nat_0) => ?ts_ph_Nat_0
19:28:44 [WARN] proved: (nmax zero ?ts_ph_Nat_0) => (nmax ?ts_ph_Nat_0 ?ts_ph_Nat_0)
19:28:44 [WARN] proved: (nmax ?ts_ph_Nat_0 ?ts_ph_Nat_0) => (nmax zero ?ts_ph_Nat_0)
19:28:44 [WARN] proved: (leq zero ?ts_ph_Nat_0) => true
19:28:44 [WARN] proved: (less ?ts_ph_Nat_0 ?ts_ph_Nat_0) => false
19:29:30 [WARN] proved: (nmax (succ ?ts_ph_Nat_0) ?ts_ph_Nat_0) => (succ ?ts_ph_Nat_0)
19:29:30 [WARN] proved: (succ ?ts_ph_Nat_0) => (nmax (succ ?ts_ph_Nat_0) ?ts_ph_Nat_0)
19:29:34 [WARN] proved: (nmax ?ts_ph_Nat_0 (succ ?ts_ph_Nat_0)) => (succ ?ts_ph_Nat_0)
19:29:34 [WARN] proved: (succ ?ts_ph_Nat_0) => (nmax ?ts_ph_Nat_0 (succ ?ts_ph_Nat_0))
19:29:50 [WARN] proved: (nmax ?ts_ph_Nat_0 (succ zero)) => (nmax (succ zero) ?ts_ph_Nat_0)
19:29:50 [WARN] proved: (nmax (succ zero) ?ts_ph_Nat_0) => (nmax ?ts_ph_Nat_0 (succ zero))
19:30:08 [WARN] proved: (less (succ ?ts_ph_Nat_0) ?ts_ph_Nat_0) => false
19:30:16 [WARN] proved: (less ?ts_ph_Nat_0 (succ ?ts_ph_Nat_0)) => true
19:30:18 [WARN] proved: (leq (succ zero) (succ ?ts_ph_Nat_0)) => true
