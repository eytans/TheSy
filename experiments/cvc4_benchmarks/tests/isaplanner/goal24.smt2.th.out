19:28:44 [0m[33m[WARN] [0mCollecting statistics
(leq ?__x0 ?__y1) => (or (= ?__x0 ?__y1) (less ?__x0 ?__y1))
(or (= ?__x0 ?__y1) (less ?__x0 ?__y1)) => (leq ?__x0 ?__y1)
(less ?x zero) => false
(less zero (succ ?x)) => true
(less (succ ?x) (succ ?y)) => (less ?x ?y)
(less ?x ?y) => (less (succ ?x) (succ ?y))
(nmax ?n ?m) => (ite (less ?n ?m) ?m ?n)
(ite (less ?n ?m) ?m ?n) => (nmax ?n ?m)
Searcher: (less ?autovar5 (succ ?autovar1))
Searcher: (less ?autovar5 (succ ?autovar3))
Searcher: (less zero ?autovar6)
Searcher: (less (succ ?autovar4) ?autovar6)
19:28:44 [0m[33m[WARN] [0m ex is: zero
19:28:44 [0m[33m[WARN] [0m ex is: (succ zero)
19:28:44 [0m[33m[WARN] [0m ex is: (succ (succ zero))
19:28:44 [0m[33m[WARN] [0mRunning exploration without proof mode, but goals were given
19:28:44 [0m[33m[WARN] [0mProof mode: true
19:28:44 [0m[33m[WARN] [0m ex is: zero
19:28:44 [0m[33m[WARN] [0m ex is: (succ zero)
19:28:44 [0m[33m[WARN] [0m ex is: (succ (succ zero))
19:28:44 [0m[33m[WARN] [0mRunning exploration without proof mode, but goals were given
19:28:44 [0m[33m[WARN] [0mProof mode: true
19:28:44 [0m[33m[WARN] [0mproved: (nmax ?ts_ph_Nat_0 ?ts_ph_Nat_0) => ?ts_ph_Nat_0
19:28:44 [0m[33m[WARN] [0mproved: (nmax zero ?ts_ph_Nat_0) => (nmax ?ts_ph_Nat_0 ?ts_ph_Nat_0)
19:28:44 [0m[33m[WARN] [0mproved: (nmax ?ts_ph_Nat_0 ?ts_ph_Nat_0) => (nmax zero ?ts_ph_Nat_0)
19:28:44 [0m[33m[WARN] [0mproved: (leq zero ?ts_ph_Nat_0) => true
19:28:44 [0m[33m[WARN] [0mproved: (less ?ts_ph_Nat_0 ?ts_ph_Nat_0) => false
19:29:30 [0m[33m[WARN] [0mproved: (nmax (succ ?ts_ph_Nat_0) ?ts_ph_Nat_0) => (succ ?ts_ph_Nat_0)
19:29:30 [0m[33m[WARN] [0mproved: (succ ?ts_ph_Nat_0) => (nmax (succ ?ts_ph_Nat_0) ?ts_ph_Nat_0)
19:29:34 [0m[33m[WARN] [0mproved: (nmax ?ts_ph_Nat_0 (succ ?ts_ph_Nat_0)) => (succ ?ts_ph_Nat_0)
19:29:34 [0m[33m[WARN] [0mproved: (succ ?ts_ph_Nat_0) => (nmax ?ts_ph_Nat_0 (succ ?ts_ph_Nat_0))
19:29:50 [0m[33m[WARN] [0mproved: (nmax ?ts_ph_Nat_0 (succ zero)) => (nmax (succ zero) ?ts_ph_Nat_0)
19:29:50 [0m[33m[WARN] [0mproved: (nmax (succ zero) ?ts_ph_Nat_0) => (nmax ?ts_ph_Nat_0 (succ zero))
19:30:08 [0m[33m[WARN] [0mproved: (less (succ ?ts_ph_Nat_0) ?ts_ph_Nat_0) => false
19:30:16 [0m[33m[WARN] [0mproved: (less ?ts_ph_Nat_0 (succ ?ts_ph_Nat_0)) => true
19:30:18 [0m[33m[WARN] [0mproved: (leq (succ zero) (succ ?ts_ph_Nat_0)) => true
done in 123424
